app/uk/gov/hmrc/exports/models/declaration/DUCR.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.exports.models.declaration
+
+import play.api.libs.json.{Json, OFormat}
+
+case class DUCR(ducr: String)
+
+object DUCR {
+  implicit val format: OFormat[DUCR] = Json.format[DUCR]
```

Comment:
this would lead to something like 
```
{
  "ducr" : {
     "ducr" : "value"
  }
}
```
in ConsignmentReferences - i really do not know do w really want to have something like this.